src/components/login/Login.scss

```
@@ -0,0 +1,20 @@
+.login {
+  display: grid;
+  place-items: center;
+  min-height: 100vh;
+  background-color: black;
+}
+
+.login > img {
+  width: 80%;
+  max-width: 1000px;
+}
+
+.login > a {
+  padding: 20px;
+  border-radius: 30px;
+  background-color: #1db954;
+  font-weight: 800;
+  color: white;
+  text-decoration: none;
+}
```

With sass we can next our selectors within each other.
```suggestion
.login {
  display: grid;
  place-items: center;
  min-height: 100vh;
  background-color: black;
  
  & > img {
  width: 80%;
  max-width: 1000px;
  }
  
  & > a {
  padding: 20px;
  border-radius: 30px;
  background-color: #1db954;
  font-weight: 800;
  color: white;
  text-decoration: none;
}
}
```